src/guides/bioinformatics/filtering-and-subsampling.rst

```diff
@@ -53,9 +53,9 @@ Subsampling within ``augur filter``
 
 Another common filtering operation is subsetting of data to a achieve a more
 even spatio-temporal distribution or to cut-down data set size to more
-manageable numbers. The filter command allows you to select a specific number of
-sequences from specific groups, for example one sequence per month from each
-country:
+manageable numbers. The filter command allows you to partition the data into
+groups based on column values and sample uniformly. For example, target one
+sequence per month from each country:
```

Comment:
I just realized that these docs don't explicitly say that the subsampling is applied to the _filtered_ data. It's buried in the `augur filter` help docs, but would be nice to reiterate here.

Reply:
Good point, I'll add more details to this doc.

Reply:
Addressed with 1b7e25f72719eee1e00de29b1c245a459533212d